src/commands/registry.rs

```diff
@@ -66,11 +72,18 @@ impl Push {
             spin_build::build(&app_file, &[]).await?;
         }
 
+        let annotations = self.annotations.map(|parameters| {
```

Comment:
The names `parameters` and `parameter` in this section are misleading.

Reply:
Ah, indeed! In the meantime, this code no longer exists. Please see the new one.